SegmentEditorSurfaceCut/SegmentEditorSurfaceCutLib/SegmentEditorEffect.py

```diff
@@ -94,6 +94,14 @@ def setupOptionsFrame(self):
 
     self.scriptedEffect.addLabeledOptionsWidget("Operation:", operationLayout)
 
+    # Smooth model checkbox layout
+    smoothModelLayout = qt.QGridLayout()
+    self.smoothModelCheckbox = qt.QCheckBox()
+    self.smoothModelCheckbox.setChecked(True) # model smoothing initial default is True
+    self.smoothModelCheckbox.setToolTip("Model is smoothed if checked, faceted if unchecked")
+    smoothModelLayout.addWidget(self.smoothModelCheckbox, 0, 0)
+    self.scriptedEffect.addLabeledOptionsWidget("Smooth model:", smoothModelLayout)

```

Comment:
If you only need to display a single widget then it should not be necessary to put it into a layout.
Have you tried to just use the checkbox directly in addLabeledOptionsWidget?

Reply:
Yes, that worked, thanks